packages/amp/src/__tests__/processors.test.tsx

```diff
@@ -65,13 +85,45 @@ test("Validate amp-iframe", async () => {
   expect(await amp(container.innerHTML, headScript)).toBeValidAmpHtml();
 });
 
+test("amp-iframe should concatenate the sandbox properties", async () => {
+  const helmetContext = {};
+
+  const { container } = render(
+    <HelmetProvider context={helmetContext}>
+      <Html2React
+        html="<iframe sandbox='allow-scripts allow-popups' src='test.html' width='auto' height='300'/>"
+        processors={processors}
+      />
+    </HelmetProvider>
+  );
+
+  expect(container.firstChild).toMatchInlineSnapshot(`
+    <amp-iframe
+      height="300"
+      layout="fixed-height"
+      sandbox="allow-scripts allow-same-origin allow-scripts allow-popups"
```

Comment:
We can concatenate the all the `allow-*` flags here and even if some property appears more than once it does not matter because it's set only once.